packages/wonder-blocks-icon/index.js

```diff
@@ -1,8 +1,8 @@
 // @flow
 import Icon from "./components/icon.js";
 import icons from "./util/icon-assets.js";
-import type {IconAsset} from "./util/icon-assets.js";
+import type {IconAsset, IconSize} from "./util/icon-assets.js";
 
-export type {IconAsset};
+export type {IconAsset, IconSize};
```

Comment:
There's a couple of places in my PR's where I noticed we need to export more things than we were.  Thankfully it's an easy fix.

Reply:
That's fair! Although if people rely upon the Flow types that means that any change to them will be a breaking change. I think we should be careful about exposing as little as we possibly can 😄